diff -rup ioquake3-1.33+svn544.orig/code/qcommon/qcommon.h ioquake3-1.33+svn544/code/qcommon/qcommon.h
--- ioquake3-1.33+svn544.orig/code/qcommon/qcommon.h	2006-02-16 12:34:38.574323552 +0100
+++ ioquake3-1.33+svn544/code/qcommon/qcommon.h	2006-02-16 12:33:48.193910895 +0100
@@ -239,7 +239,10 @@ extern int demo_protocols[];
 #define MASTER_SERVER_NAME	"master.quake3arena.com"
 #endif
 #ifndef AUTHORIZE_SERVER_NAME
-#define	AUTHORIZE_SERVER_NAME	"authorize.quake3arena.com"
+//  we set the authorize server to localhost. it will challenge twice
+// and let the clients in anyway.
+// #define	AUTHORIZE_SERVER_NAME	"authorize.quake3arena.com"
+#define	AUTHORIZE_SERVER_NAME	"localhost"
 #endif
 
 #define	PORT_MASTER			27950
