Index: ioquake3-1.34-rc1+svn708/code/qcommon/q_shared.h
===================================================================
--- ioquake3-1.34-rc1+svn708.orig/code/qcommon/q_shared.h	2006-04-16 08:35:49.000000000 +0200
+++ ioquake3-1.34-rc1+svn708/code/qcommon/q_shared.h	2006-04-16 08:42:26.346950987 +0200
@@ -26,7 +26,11 @@
 // q_shared.h -- included first by ALL program modules.
 // A user mod should never modify this file
 
-#define Q3_VERSION            "ioq3 1.33"
+// #define Q3_VERSION            "ioq3 1.33"
+// The upstream version has been put down again, we don't change this
+// as it would create confusion for the packages, the 1.34 is bound to
+// arrive soon anyway.
+#define Q3_VERSION            "GNU/Debian ioQ3 1.34-rc1"
 #define CLIENT_WINDOW_TITLE   "icculus.org/quake3"
 #define CLIENT_WINDOW_ICON    "ioq3"
 #define CONSOLE_WINDOW_TITLE  "icculus.org/quake3 console"
Index: ioquake3-1.34-rc1+svn708/code/server/sv_init.c
===================================================================
--- ioquake3-1.34-rc1+svn708.orig/code/server/sv_init.c	2006-04-16 08:35:45.000000000 +0200
+++ ioquake3-1.34-rc1+svn708/code/server/sv_init.c	2006-04-16 08:40:38.195010759 +0200
@@ -579,7 +579,7 @@
 	Cvar_Get ("protocol", va("%i", PROTOCOL_VERSION), CVAR_SERVERINFO | CVAR_ROM);
 	sv_mapname = Cvar_Get ("mapname", "nomap", CVAR_SERVERINFO | CVAR_ROM);
 	sv_privateClients = Cvar_Get ("sv_privateClients", "0", CVAR_SERVERINFO);
-	sv_hostname = Cvar_Get ("sv_hostname", "noname", CVAR_SERVERINFO | CVAR_ARCHIVE );
+	sv_hostname = Cvar_Get ("sv_hostname", "GNU/Debian server", CVAR_SERVERINFO | CVAR_ARCHIVE );
 	sv_maxclients = Cvar_Get ("sv_maxclients", "8", CVAR_SERVERINFO | CVAR_LATCH);
 
 	sv_maxRate = Cvar_Get ("sv_maxRate", "0", CVAR_ARCHIVE | CVAR_SERVERINFO );
